reconcile/typed_queries/app_interface_vault_settings.py

```diff
@@ -10,19 +11,27 @@
 from reconcile.utils import gql
 
 
-def get_app_interface_vault_settings_optional() -> Optional[AppInterfaceSettingsV1]:
+def get_app_interface_vault_settings_optional(
+    query_func: Optional[Callable] = None,
```

Comment:
Thanks for the cleanup here :bow: :heart:

Reply:
I wanted to add a test for it and this was the best way I found to do it 😄